index.html

```diff
@@ -9,21 +9,23 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     
     <link rel="stylesheet" href="./css/styles.css">
+    
     <title>Mariane | Professional Web Developer</title>
 </head>
 <body>
    <section class="root-container">
-    <div class="header-section">
+    <div class="header-section" id="header-section">
         <header>
-            <nav class="nav-link">
+            <nav>
                 <a href="#">My Logo</a>
-                <ul class="navigation-bar">
-                    <li><a href="#" class="nav-link">Portfolio</a></li>
-                    <li><a href="#" class="nav-link">About</a></li>
-                    <li><a href="#" class="nav-link">Contact</a></li>
+                <ul id="navbar">
+                    <li><a href="javascript:void(0)" class="closebtn">&times;</a></li>
+                    <li><a href="#portfolio" class="nav-link">Portfolio</a></li>
+                    <li><a href="#about-me" class="nav-link">About</a></li>
+                    <li><a href="#contact" class="nav-link">Contact</a></li>
                 </ul>

```

Comment:
You have done a great job with your mobile portfolio menu. However, according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), you are supposed to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). You are currently left with the following styling changes to make your project perfect and ready for approval:

- [ ] Currently, your menu design does not match the chosen Figma design. When I tap the hamburger bars, it is supposed to cover the entire page of the screen as it is shown in the Figma design. The choice of colors you currently have also does not match the Figma design. You have a white background and it is supposed to be a bluish color as provided in the Figma design. Kindly fix this.

**This is what your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/181208118-c36d24fc-e8a8-4936-9e92-88d3ae3b5f66.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/181208306-2d88839b-2b6f-4952-863d-6cbad5d463a3.png)

- [ ] When I click on the hamburger bars, the bars as well as the logo are supposed to be hidden from view. Currently they are still visible. Kindly fix this.

**This is what your menu heading currently loos like:**

![image](https://user-images.githubusercontent.com/86318284/181208796-6160da5a-c9ae-45bd-9d90-744c3a843291.png)

**This is the desired Figma design menu header:**

![image](https://user-images.githubusercontent.com/86318284/181209063-0122741b-0367-4a8f-b8c9-8ca9c4777f4e.png)

- [ ] Also, kindly note that the portfolio nav links are supposed to be placed at the left side of the screen. Kindly do so when you do the styling.